src/views/NewQuestion/Sections/SectionForm.tsx

```diff
@@ -211,15 +210,16 @@ const SectionForm = (props: RouteComponentProps) => {
     });
   };
 
-  const formControls = () => {
+  const isInvalidForm = () => {
     return (
       email.length < 1 ||
```

Comment:
Kan godt ta bort denne linjen også, da den blir plukket opp av `isValidEmail`

Reply:
Yes! Done in 651f4db